draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -227,10 +227,19 @@ Properties are represented by RRsets below the corresponding member node.
 ## The Group Property
 
 The property is defined by a TXT record in the sub-node labelled `group`.
+By using this property, the producer can signal the consumer(s) to treat some member zones within the catalog zone differently.
+The consumer MAY apply different configuration options when configuring the member zones, based on the group property value.
```

Comment:
It's unclear whether `configuring` refers to "setting stuff within the catalog zone, for this member" vs. "configuring the DNS server with the zone". I think the latter is meant.

Suggestion: `configuring the` --> `processing`